### DISCUSSION TITLE
"no such file or directory" troubles.

### DISCUSSION THREAD

**Discussion by nradina** [pschatzmann/arduino-audio-tools#519] [Q&A] [answered]
Hi,

I'm attempting to try out some of these wonderful examples yet encountering the "no such file or directory" error in relation to the  "#include "AudioTools.h" line.

I chose the "ESP32 Dev Module" as the board.
I loaded the sketchbook example for the "streams-i2s-serial".
I double checked the library path and also the actual example .ino file. All looks good.
The ESP32 library is 1.0.6
Arduino IDE version 1.8.16 (MAC)

I'm sure this is my own user error, yet can't seem to work out a solution on my own.

Thanks for any guidance.

-------error message------
Arduino: 1.8.16 (Mac OS X), Board: "ESP32 Dev Module, Disabled, Default 4MB with spiffs (1.2MB APP/1.5MB SPIFFS), 240MHz (WiFi/BT), QIO, 80MHz, 4MB (32Mb), 921600, None"

streams-i2s-serial:11:24: fatal error: AudioTools.h: No such file or directory
compilation terminated.
exit status 1
AudioTools.h: No such file or directory

**Comment by pschatzmann** [comment, marked as answer]
Something must have gone wrong with the installation of this library. I suggest to google the topic how to install Arduino libraries from Github.
In the Arduino Properties you can find the location of the libraries directory. After a successfull installation you should have the following directory structure
- libraries
  - arduino-audio-tools
    - src
       - AudioTools.h
       - many other
